lib/w_common.dart

```diff
@@ -12,7 +12,13 @@
 // See the License for the specific language governing permissions and
```

Comment:
Actually, since we're renaming VM to common, I'll just delete the vm one and we'll end up with `w_common` and `w_common_browser`. Let me know if you disagree.